server.php

```diff
@@ -128,23 +128,40 @@ function valet_default_site_path($config)
 function get_valet_site_path($valetConfig, $siteName, $domain)
 {
     $valetSitePath = null;
+
     foreach ($valetConfig['paths'] as $path) {
-        if ($handle = opendir($path)) {
-            while (false !== ($file = readdir($handle))) {
-                if (! is_dir($path.'/'.$file)) continue;
-                if (in_array($file, ['.', '..', '.DS_Store'])) continue;
-
-                // match dir for lowercase, because Nginx only tells us lowercase names
-                if (strtolower($file) === $siteName || strtolower($file) === $domain) {
-                    $valetSitePath = $path.'/'.$file;
-                }
+        $handle = opendir($path);
+
+        if ($handle === false) {
+            continue;
+        }
+
+        $dirs = [];
+
+        while (false !== ($file = readdir($handle))) {
+            if (is_dir($path.'/'.$file) && ! in_array($file, ['.', '..', '.DS_STORE'])) {
```

Comment:
`.DS_STORE` should go back to `. DS_Store`

Reply:
It looks like the current master version of server.php says `.DS_STORE` without the space, is this incorrect?

Reply:
Excuse me, I misunderstood :) fixed it

Reply:
Correct. My bad with sloppy copy/paste 🤣

Reply:
I suspect the space is a typo, but it probably shouldn't all be capitalized.

Reply:
Looks properly corrected now. 👍

Reply:
Looking into this further, I wonder if we need this check at all, since the `is_dir` check comes first, which returns `false` for `.DS_Store`?

Reply:
True: `.DS_Store` is a file, not a directory.

Reply:
If you're gonna remove it, do you mind rolling back your local working branch to the Flattening commit, before removing it? And then force-push the commit to the PR, to keep the history cleaner?

Reply:
Sure thing! Squashed them together :)